integration-tests/testnet-scripts/start-docker.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# If -e is not set then if the build fails, it will use the old container, resulting in a very confusing debugging situation
+# Setting -e makes it error out if the build fails
+set -eux 
+
+CONTAINER_NAME=$1
+INSTANCE_NAME=$2
+LOCAL_SDK_PATH=${3:-"default"} # Sets this var to default if null or unset
+
+# Remove existing container instance
+set +e
+docker rm -f "$INSTANCE_NAME"
+set -e
+
+# Delete old sdk directory if it exists 
+if [ -d "./cosmos-sdk" ]; then
+    rm -rf ./cosmos-sdk/
+fi 
+
+# Copy sdk directory to working dir if path was specified
+if [[ "$LOCAL_SDK_PATH" != "default" ]]
+then
+    cp -n -r "$LOCAL_SDK_PATH" ./cosmos-sdk
```

Comment:
This implementation copies the local sdk repo into the ICS repo, pipes it to docker, then deletes the copy. There are alternatives to this scheme, but the chosen solution seemed the simplest and standardizable.

We could also try building the docker image outside the ICS repo to allow ```ADD``` commands with external directories

Reply:
love the `cp -n` 👍